app/workers/import_worker.rb

```diff
@@ -35,6 +37,14 @@ def import_rows
     CSV.new(import_contents).reject(&:blank?)
   end
 
+  def process_mutes
+    import_rows.each do |row|
+      target_account = Account.find_local(row.first)
```

Comment:
Ah, I hadn't thought about it that way.

I had thought that a person bringing an old mutes list from a former instance with them to a new instance would probably not need to worry about accounts that the new instance doesn't know about ... but you're right, we want to proactively mute them ahead of time in case the new instance learns about them.

Will update to match block behavior.